src/sentry/static/sentry/app/components/organizations/headerItem.jsx

```diff
@@ -14,18 +14,22 @@ class HeaderItem extends React.Component {
      * className for <Label> component
      */
     labelClassName: PropTypes.string,
+    margin: PropTypes.string,
```

Comment:
Let's get rid of this prop and just use `-5px` for the margin. We'll make it look the same in Health later.

Reply:
So get rid of the `space(1)` all together?